### PR TITLE
feat:Add baremetal cluster e2e test

### PR DESCRIPTION
**Pull request by confoc**
Add new baremetal database CRUD

### CODE REVIEW

File: tests/e2e/greptimedbcluster_test.go [hunk 1]
```diff
@@ -75,6 +75,21 @@ var _ = Describe("Basic test of greptimedb cluster", func() {
 		err = listCluster()
 		Expect(err).NotTo(HaveOccurred(), "failed to list cluster")
 
+		go func() {
+			err = createClusterinBaremetal()
+			Expect(err).NotTo(HaveOccurred(), "failed to create cluster in baremetal")
+		}()
+
+		go func() {
+			time.Sleep(100 * time.Second)
```
Comment (daviderli614) [inline tests/e2e/greptimedbcluster_test.go]: suggestion: It is not recommended to use the sleep. You can modify it to periodically detect whether the port is connected and set a timeout.

**Comment by daviderli614** [comment]
suggestion：The k8s mode and baremetal mode should be located in different places for testing, not in the same place.

**Comment by daviderli614** [comment]
This [action](https://github.com/GreptimeTeam/gtctl/actions/runs/9544957630/job/26304766329) has following errors, but the test passes?
```
component 'frontend.0' binary '/home/runner/.gtctl/artifacts/binaries/greptime/v0.8.2/bin/greptime' (pid '17288') exited with error: exit status 1
args: '[--log-level=info frontend start --metasrv-addr=0.0.0.0:3002 --http-addr=0.0.0.0:4000 --rpc-addr=0.0.0.0:4001 --mysql-addr=0.0.0.0:4002 --postgres-addr=0.0.0.0:4003]'
```

**Comment by zyy17** [comment]
@confoc Are you still working on it?

**Comment by zyy17** [comment]
@confoc Close the PR for inactive development. If you want to continue it in the future, you can re-open the PR or create a new PR.

**Pull request closed without merging**